scripts/zkai_stwo_statement_only_attempt_transcript_gate.py

```
@@ -0,0 +1,812 @@
+        "nanozk_gap_typed_bytes": 32_616,
+    }
+    if summary != expected_summary:
+        raise StatementOnlyAttemptTranscriptGateError("binding summary drift")
+    if tuple(payload.get("non_claims", ())) != NON_CLAIMS:
+        raise StatementOnlyAttemptTranscriptGateError("non_claims drift")
+    if tuple(payload.get("validation_commands", ())) != VALIDATION_COMMANDS:
+        raise StatementOnlyAttemptTranscriptGateError("validation command drift")
+    if check_mutations:
+        mutation = _dict(payload.get("mutation_result"), "mutation result")
+        if mutation.get("rejected") != len(MUTATION_NAMES) or mutation.get("accepted") != 0:
+            raise StatementOnlyAttemptTranscriptGateError("mutation summary drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)